### ISSUE TITLE
6 - Transformers for Sentiment Analysis  Model class typo

### ISSUE THREAD

**Issue by ChrisProgramming2018** [bentrevett/pytorch-sentiment-analysis#63]
I think in the forward function of the  Model class 
`
def forward(self, text):
        
        #text = [batch size, sent len]
                
        with torch.no_grad():
            embedded = bert(text)[0]
                
        #embedded = [batch size, sent len, emb dim]
        
        _, hidden = self.rnn(embedded)
`
bert(text)[0] needs to be replaced with self.bert(text)[0]

**Comment by bentrevett** [comment]
You're correct, will change that now.
